gpflow/optimizers/mcmc.py

```diff
@@ -109,13 +102,13 @@ def grad_fn(dy, variables: Optional[tf.Tensor] = None):
 
     def convert_constrained_values(self, hmc_samples):
         """
-        Converts list of `unconstrained_values` to constrained versions. Each value in the list correspond to an entry in
-        `self.parameters`; in case that object is a `gpflow.Parameter`, the `forward` method of its transform
-        will be applied first.
+        Converts list of `unconstrained_values` to constrained versions. Each value in the
```

Comment:
was this formatting done with black and line-length 80?

Reply:
Nope